ansible/roles/jenkins-worker/templates/freebsd.initd.j2

```diff
@@ -28,8 +28,8 @@ jenkins_env=" \
   NODE_TEST_DIR=/home/{{ server_user }}/tmp \
   PATH=/usr/local/libexec/ccache:/usr/local/bin:${PATH} \
   JOBS={{ server_jobs|default(ansible_processor_vcpus|default('2')) }} \
-  CC=cc \
-  CXX=c++"
+  CC={{ custom_cc_command|default('cc') }} \
```

Comment:
@kfarnung Thanks for elaborating on your thoughts; perhaps the config file is the better way - my rationale against it is that having it in the config file isn't alone "enough" to install and pass to CC -- if that is the case we would have to change the variables as well.